ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -137,6 +158,39 @@ da_scala_binary(
     ],
 )
 
+da_scala_binary(
+    name = "conformance-test-oracle-bin",
```

Comment:
Nit. Can we call it something else other than "conformance-test-*"? It is not a test but an executable that we can use i a test. So a name such as "sandbox-on-x-with-oracle-bin" would be more fitting. I worry about text searches for "conformance-test" that will lead us astray.
Same applies to the following target

Reply:
Actually these binaries surround themselves with DB fixtures, and only used for conformance tests. Also aligned to what we had already in ledger-on-sql. From my side these are good names, but open for suggestions.